pam/pam.go

```diff
@@ -74,7 +76,8 @@ func (h *pamModule) Authenticate(mTx pam.ModuleTransaction, flags pam.Flags, arg
 	client, closeConn, err := newClient(args)
 	if err != nil {
 		log.Debug(context.TODO(), err)
-		return pam.ErrAuthinfoUnavail
+		_ = showPamMessage(mTx, pam.ErrorMsg, err.Error())
```

Comment:
Can you add a comment here explaining why the error does not matter? (or why it never errors, if so) 
We don't care about it in tests, but I think it's important to have those in production code.

Reply:
I'm fine with just logging it or joining it with the other one, but not being able to show the message is a relevant thing to report.

Ps.: I saw that you decided to log it, so it's fine by me.

Reply:
Yeah, joining is a bit dangerous I think, since it may override the other pam.Error we're setting? Not sure, but better not to risk :)